example/email_action_delivered.json

```diff
@@ -0,0 +1,31 @@
+{
+  	"meta": {
+		"topic": [
+			"marketing"
+		]
+	},
+	"type": "DeliveryEvent",
```

Comment:
Are you reporting the email being sent or accepted by the mail server?  In either case, you can use SendAction and the key 'actionStatus' to express a successful (CompletedActionStatus) or unsuccessful (FailedActionStatus) result.

Reply:
Ok will do